### PR TITLE
Added supporters and affiliates section in frontpage

### PR DESCRIPTION
**Pull request by AnXh3L0**
Preview
Extra of #215 

image

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 gem 'github-pages', group: :jekyll_plugins
 gem 'tzinfo-data'
 gem 'wdm', '>= 0.1.0' if Gem.win_platform?
+gem 'jekyll-commonmark-ghpages'
```
Comment (RDIL) [inline Gemfile]: Included in GHP gem

File: _sass/_front_page.scss [hunk 2]
```diff
@@ -235,3 +235,17 @@ ul.banner-social-buttons {
...
+  text-align: center;
+  padding-bottom: 40px;
+  padding-top: 40px;
+
+  & img {
+    margin: 25px 25px;
+  }
+}
+
+/*** Affiliates Section ***/
```
Comment (RDIL) [inline _sass/_front_page.scss]:
```suggestion
```

File: _sass/_front_page.scss [hunk 3]
```diff
@@ -235,3 +235,17 @@ ul.banner-social-buttons {
...
+.supporters, .affiliates {
+  text-align: center;
+  padding-bottom: 40px;
+  padding-top: 40px;
+
+  & img {
+    margin: 25px 25px;
+  }
+}
+
```
Comment (RDIL) [inline _sass/_front_page.scss]:
```suggestion
```

File: _sass/_front_page.scss (hunk 2 above)
Comment (RDIL) [inline _sass/_front_page.scss]: Minify to reduce loading times please

Comment (RDIL) [reply _sass/_front_page.scss]: (The xml sorry wrong place)

File: index.html [hunk 4]
```diff
@@ -61,6 +61,32 @@ <h2>Activity</h2>
...
+              <object type="image/svg+xml" data="https://opencollective.com/opensourcedesign/tiers/sponsor.svg?avatarHeight=50&width=400"></object>
+              <object type="image/svg+xml" data="https://opencollective.com/opensourcedesign/tiers/backer.svg?avatarHeight=50&width=400"></object>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-sm-12 affiliates">
+            <h2>Affiliates</h2>
+              <img src="images/supporters/eff.svg" height="40px" width="auto" alt="Electronic Frontier Foundation Logo"/>
+              <img src="images/supporters/fsfe.svg" height="50px" width="auto" alt="Free Software Foundation Europe Logo"/>
+              <a href="https://opensource.org/affiliates" target="_blank" rel="noopener"><img src="images/supporters/osi.svg" height="100px" width="auto" alt="Open Source Initiative Logo"/></a>
```
Comment (RDIL) [inline index.html]: Title attr on anchor tag missing

**Comment by AnXh3L0** [comment]
@RDIL and @elioqoshi if you can approve this please so we can merge it :)

**Comment by jancborchardt** [comment]
Looking super sharp, thanks @AnXh3L0! :)

**Pull request merged**